docs/ops/comparison/NotEqual_1.md

```diff
@@ -4,7 +4,18 @@
 
 **Category**: Comparison binary operation
 
-**Short description**: *NotEqual* performs element-wise comparison operation with two given tensors applying multi-directional broadcast rules.
+**Short description**: *NotEqual* performs element-wise comparison binary operation with two given tensors applying
+multi-directional broadcast rules.
```

Comment:
`binary` is extra. We don't use this word for similar operation description. Please remove it